gcn_monitor/kafka.py

```diff
@@ -44,3 +53,5 @@ def run():
                 log.info("topic %s: got message", topic)
                 partition = message.partition()
                 metrics.received.labels(topic, partition).inc()
+                key = f"{topic}-{datetime.now(timezone.utc)}"
```

Comment:
- Topic names may contain characters that are not valid in filenames. We will need to reversibly escape them.
- Let's organize records into folders by topic.
- Is there a maximum number of files within a directory? If so, we may need to chunk the filenames.
- Is the expression `{datetime.now(timezone.utc)}` guaranteed to be unique for each Kafka record?

Reply:
I think I got all of these in the latest commit. As for the last 2 points:
- From https://aws.amazon.com/s3/faqs/ I believe there is no size limit to files in a directory
- Removed the datetime thing, there is already a timestamp on the message, and it is now included in the `put_object` call